test/image/png_indexed/test_png_indexed.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+
+import fpdf
+from test.conftest import assert_pdf_equal
+from PIL import Image
+
+
+HERE = Path(__file__).resolve().parent
+
+
+def test_png_indexed_no_transparency(tmp_path):
+    pdf = fpdf.FPDF()
+    pdf.add_page()
+
+    # P images
+    assert Image.open(HERE / "flower1.png").mode == "P", "img.mode is not P"
+
+    pdf.set_image_filter("FlateDecode")
+    pdf.image(HERE / "flower1.png", x=10, y=10, w=50, h=50)
```

Comment:
You have really beautiful flowers in your garden ^^
Thanks for sharing!

Reply:
Thank you!